environments/india/public.yml

```diff
@@ -132,6 +132,7 @@ localsettings:
   ENABLE_SOFT_ASSERT_EMAILS: True
   ENFORCE_SSO_LOGIN: True
   ELASTICSEARCH_MAJOR_VERSION: 2
+  ES_CASE_SEARCH_INDEX_NAME: "case_search_2022-10-14"
```

Comment:
this should also be "case_search_2022-10-04" correct?

Reply:
no, I copied the name from the doc when I created this index (and the doc has this name)

Reply:
I must have made a typo when I created the one on prod